packages/grafana-data/src/panel/registryFactories.ts

```diff
@@ -66,6 +71,16 @@ export function createFieldConfigRegistry<TFieldConfigOptions>(
       }
     }
 
+    if (fieldConfigProp.id === FieldConfigProperty.Mappings) {
+      fieldConfigProp = {
+        ...fieldConfigProp,
+        settings: {
+          ...fieldConfigProp.settings,
+          enableUncommonOptions: config.enableUncommonOptions,
+        },
+      };
+    }
```

Comment:
Think this could be a mistake that just complicates things (with a special rule), easier to just require setting for FieldConfigProperty.Mappings that control if the panel support value icons. These things seem not 100% correlated, just because a panel support icon on the field level does not mean you support it on the value level. 

Sure it will mean that panels that do support both have to enable both the field config (explicitAdd) and icon support for value mappings. Something like:


```ts
standardOptions: {
      [FieldConfigProperty.Icon]: {
         explicitAdd: true
      },
      [FieldConfigProperty.Mappings]: {
         settings: {
            icon: true
         }  
      },
    },
```